package.json

```diff
@@ -76,15 +77,17 @@
     "classnames": "^2.5.1",
     "color": "^4.2.3",
     "color.js": "^1.2.0",
-    "date-fns": "^3.5.0",
+    "date-fns": "^3.6.0",
     "electron-squirrel-startup": "^1.0.0",
     "flexsearch": "^0.7.43",
+    "got-scraping": "^4.0.5",
     "i18next": "^23.10.1",
     "i18next-browser-languagedetector": "^7.2.0",
     "jsdom": "^24.0.0",
     "lodash": "^4.17.21",
-    "parse-torrent": "9.1.5",
+    "parse-torrent": "^9.1.5",
```

Comment:
Please keep the `pase-torrent` version locked due to the low compatibility of newer versions on `win32`